receiver/hostmetricsreceiver/hostmetrics_receiver.go

```diff
@@ -172,7 +174,9 @@ func (hmr *receiver) scrapeMetrics(ctx context.Context) {
 	}
 
 	if len(errors) > 0 {
-		span.SetStatus(trace.Status{Code: trace.StatusCodeDataLoss, Message: fmt.Sprintf("Error(s) when scraping metrics: %v", componenterror.CombineErrors(errors))})
+		combinedErrs := componenterror.CombineErrors(errors)
+		hmr.logger.Error("Error(s) when scraping metrics", zap.Error(combinedErrs))
```

Comment:
Why this change?

Reply:
This branch is out of date. This file no longer exists and the trace reporting was moved to `obsreport_scraper.go`